packages/block-library/src/media-text/edit.native.js

```diff
@@ -131,18 +132,22 @@ class MediaTextEdit extends Component {
 			attributes,
 			backgroundColor,
 			setAttributes,
-			// isMobile,
+			isMobile,
+			isSelected,
+			isParentSelected,
+			isAncestorSelected,
 		} = this.props;
 		const {
-			// isStackedOnMobile,
+			isStackedOnMobile,
 			mediaPosition,
 			mediaWidth,
 			verticalAlignment,
 		} = attributes;
-		const shouldStack = false; // We are temporarily not stacking until we fix alignment buttons
-		// const shouldStack = isStackedOnMobile && isMobile; // <<< Original line
+		const shouldStack = isStackedOnMobile && isMobile;
 		const temporaryMediaWidth = shouldStack ? 100 : ( this.state.mediaWidth || mediaWidth );
 		const widthString = `${ temporaryMediaWidth }%`;
+
+		const selectedStyle = ! shouldStack && { paddingRight: 0, paddingLeft: 0, ...( mediaPosition === 'right' && styles.innerPaddingMediaOnRight ), ...( mediaPosition === 'left' && styles.innerPaddingMediaOnLeft ) };
```

Comment:
You could move `{ paddingRight: 0, paddingLeft: 0 }` to styles

Reply:
done